app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
+
+  def edit_terms
+    if(current_user != nil)
```

Comment:
This project is using `cancancan` and `devise` - I think there are better ways to ensure the user is not nil or that the user has to be authorized to perform this action. 
This of course will work but it's not the cleanest solution.

Also, you could use `.nil?` and you don't need to find the user when you have `current_user` available in this context - it already has found the user for you.

Reply:
I must say i need to look more on cancancan ....

i was repair nil and remove finding user.